streamlit-java/.gitignore

```diff
@@ -1,2 +1,3 @@
 snowflake.local.yml
 output/**
+target
```

Comment:
This is output from the java build step?

Reply:
yes